hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/io/erasurecode/CodecUtil.java

```diff
@@ -170,8 +174,14 @@ private static String[] getRawCoderNames(
 
   private static RawErasureEncoder createRawEncoderWithFallback(
       Configuration conf, String codecName, ErasureCoderOptions coderOptions) {
+    boolean ISALEnabled = conf.getBoolean(IO_ERASURECODE_CODEC_NATIVE_ENABLED_KEY,
+        IO_ERASURECODE_CODEC_NATIVE_ENABLED_DEFAULT);
     String[] rawCoderNames = getRawCoderNames(conf, codecName);
     for (String rawCoderName : rawCoderNames) {
+      if (!ISALEnabled && rawCoderName.contains("native")) {
+        LOG.info("Disable the encoder with ISA-L.");
```

Comment:
Is this more appropriate as debug-level logging? I'm wondering if at info level there could be problems of 1) the message accidentally leaking to user-facing output if this code path is triggered as part of commands like `hdfs dfs -cat` on erasure-coded files, and 2) excessive logging if this is triggered many times.

Reply:
@cnauroth Thank you for your suggestion. I'v updated the code and changed the log level to debug. Could you review this please?